signinwithapplebutton/src/main/java/com/willowtreeapps/signinwithapplebutton/SignInWithAppleService.kt

```diff
@@ -3,15 +3,35 @@ package com.willowtreeapps.signinwithapplebutton
 import android.net.Uri
 import android.os.Parcel
 import android.os.Parcelable
+import android.view.View
+import androidx.annotation.IdRes
 import androidx.fragment.app.FragmentManager
+import com.willowtreeapps.signinwithapplebutton.view.SignInWebViewDialogFragment
 import java.util.*
 
 class SignInWithAppleService(
-    private val clientId: String,
-    private val redirectUri: String,
-    private val scope: String
+    @IdRes private val id: Int,
+    private val fragmentManager: FragmentManager,
+    private val config: SignInWithAppleConfig,
+    private val callback: (SignInWithAppleResult) -> Unit
 ) {
 
+    constructor(
+        @IdRes id: Int,
+        fragmentManager: FragmentManager,
+        config: SignInWithAppleConfig,
+        callback: SignInWithAppleCallback
+    ) : this(id, fragmentManager, config, callback.toFunction())
+
+    init {
+        val fragmentIfCreated =
+            fragmentManager.findFragmentByTag(fragmentTag) as? SignInWebViewDialogFragment
+        fragmentIfCreated?.configure(callback)
```

Comment:
I think if we rename the button's configure method to "set up", we should change this configure method as well.

Reply:
Mm, maybe it's different since it's external to the object now, rather than being a required configuration injection method.